wordpress-mobile/WordPress-iOS
# Feat: Multiple selection of media in gutenberg

In this PR I added the possibility to set multiple media when inserting to the block.
Related `gutenberg-mobile` PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/1331
To test:
At this moment we don't support `gallery block` so this change can not be tested without changes in `gutenberg-mobile` JS bundle. In the future, we will use this functionality to multiple selections of media for the gallery.


* set multiple prop to true in media-upload bloc
* add image/video block - multiple select should be enabled for selecting media from the device or WP Media
* the array of selected media should be passed to the callback function
